app/views/transactions/index.html.erb

```diff
@@ -3,39 +3,45 @@
 
   <%= render partial: "transactions/summary", locals: { totals: @totals } %>
 
-  <div id="transactions" data-controller="bulk-select" data-bulk-select-resource-value="<%= t(".transaction") %>" class="bg-white rounded-xl border border-alpha-black-25 shadow-xs p-4 space-y-4">
+  <div id="transactions" data-controller="bulk-select" data-bulk-select-resource-value="<%= t(".transaction") %>" class="bg-white rounded-xl border border-alpha-black-25 shadow-xs pb-4">
+
+    <div class="p-4 pb-0">
+      <%= render partial: "transactions/searches/search", locals: { transactions: @transactions } %>
+    </div>
 
-    <%= render partial: "transactions/searches/search", locals: { transactions: @transactions } %>
 
     <% if @transactions.present? %>
       <div hidden id="transaction-selection-bar" data-bulk-select-target="selectionBar">
         <%= render "selection_bar" %>
       </div>
-
-      <div class="grid grid-cols-12 bg-gray-25 rounded-xl px-5 py-3 text-xs uppercase font-medium text-gray-500 items-center mb-4">
-        <div class="pl-0.5 col-span-4 flex items-center gap-4">
-          <%= check_box_tag "selection_transaction",
-                            class: "maybe-checkbox maybe-checkbox--light",
-                            data: { action: "bulk-select#togglePageSelection" } %>
-          <p class="col-span-4">transaction</p>
+      <div class="h-[calc(100dvh-370px)] overflow-y-auto px-4 border-y border-alpha-black-100">
```

Comment:
In regards to the calculated height, I think an improvement we can make to this layout is to use flex columns to dynamically size this.  This will require a few small tweaks to the overall layout, but I think it will be nice to have everything relative.  Here's the diff that I used to get it working:

```diff
diff --git a/app/views/layouts/with_sidebar.html.erb b/app/views/layouts/with_sidebar.html.erb
index f014dd6..ca824d4 100644
--- a/app/views/layouts/with_sidebar.html.erb
+++ b/app/views/layouts/with_sidebar.html.erb
@@ -7,7 +7,7 @@
         <%= render "layouts/sidebar" %>
       <% end %>
     </div>
-    <main class="grow px-20 pt-6 pb-32 h-full overflow-y-auto">
+    <main class="grow px-20 py-6 h-full overflow-y-auto">
       <%= yield %>
     </main>
   </div>
diff --git a/app/views/transactions/index.html.erb b/app/views/transactions/index.html.erb
index 5fe827d..ff40b02 100644
--- a/app/views/transactions/index.html.erb
+++ b/app/views/transactions/index.html.erb
@@ -1,20 +1,19 @@
-<div class="space-y-4">
+<div class="space-y-4 h-full flex flex-col overflow-y-auto">
   <%= render "header" %>
 
   <%= render partial: "transactions/summary", locals: { totals: @totals } %>
 
-  <div id="transactions" data-controller="bulk-select" data-bulk-select-resource-value="<%= t(".transaction") %>" class="bg-white rounded-xl border border-alpha-black-25 shadow-xs pb-4">
+  <div id="transactions" data-controller="bulk-select" data-bulk-select-resource-value="<%= t(".transaction") %>" class="grow overflow-y-auto flex flex-col bg-white rounded-xl border border-alpha-black-25 shadow-xs pb-4">
 
     <div class="p-4 pb-0">
       <%= render partial: "transactions/searches/search", locals: { transactions: @transactions } %>
     </div>
 
-
     <% if @transactions.present? %>
       <div hidden id="transaction-selection-bar" data-bulk-select-target="selectionBar">
         <%= render "selection_bar" %>
       </div>
-      <div class="h-[calc(100dvh-370px)] overflow-y-auto px-4 border-y border-alpha-black-100">
+      <div class="grow overflow-y-auto px-4 border-alpha-black-100">
         <div class="grid grid-cols-12 bg-gray-25 rounded-xl px-5 py-3 text-xs uppercase font-medium text-gray-500 items-center mb-4">
           <div class="pl-0.5 col-span-4 flex items-center gap-4">
             <%= check_box_tag "selection_transaction",
```